src/components/home/slider/index.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react'
+import { useEffect, FC } from 'react'
+import SliderChildren from './slider'
+import { useSetState } from '../../hooks/useSetState';
+import { Movie } from './sliderModal'
+
+interface Props {
+  loading: boolean;
+  movies: Movie[];
+}
+
+const Slider: FC<Props> = ({ movies, loading }) => {
+  const [state, setState] = useSetState({
+    activeIndex: 2,
+    translateValue: 0
+  })
+
+  const nextImg = () => {
+    setState(
+      state.activeIndex === movies.length - 1
+        ? {
+          activeIndex: 2,
+          translateValue: 0,
+        }
+        : {
+          activeIndex: state.activeIndex + 1,
+          translateValue: state.translateValue - 250,
+        }
+    )
+  }
+
+  const backImg = () => {
+    setState(
+      state.activeIndex === 0
+        ? null
+        : {
+          activeIndex: state.activeIndex - 1,
+          translateValue: state.translateValue + 250,
+        }
+    )
+  }
+
+
+  const handleArrow = (e: KeyboardEvent) => {
+    console.log(true)
```

Comment:
`console.log`s are not good to have in production code. I suggest deleting.

Reply:
Removed it.

Reply:
Done  :white_check_mark: